main.js

```diff
@@ -5,4 +5,137 @@ const menuToggle = document.getElementById('collapsibleNavbar');
 const bsCollapse = new bootstrap.Collapse(menuToggle);
 navLinks.forEach((n) => {
   n.addEventListener('click', () => { bsCollapse.toggle(); });
```

Comment:
- [ **OPTIONAL**] As pointed out by the previous reviewer, I think it would be better if your **mobile popup menu** is not set up to be displayed by default when changing to **Mobile version** to make your portfolio more professional. Please, I suggest you set it to be displayed when the user clicks the Humbarger button. I know you can do that :muscle: :muscle: